packages/api/tests/helpers/create-test-data.js

```diff
@@ -70,23 +66,10 @@ async function createGranuleAndFiles({
 
   const collectionName = randomString(5);
   const collectionVersion = randomString(3);
-
-  // If a collectionId for a Dynamo Collection was not passed,
```

Comment:
This is unneeded for unit testing as dynamo has no database integrity constraints.